0800-appendix-198215-dale.Rmd

```diff
@@ -153,9 +148,9 @@ m^2^/\$1000.
 
 ## Conclusion {.unnumbered}
 
-The BC freshwater atlas streamline indicates the stream gradient of the segment upstream of the FSR was 7.5% which coincides approximately with stream measurements.  This value is very near the 7.4% cutoff for modelling steelhead high value rearing habitat so we estimated 0.8km of habitat suitable for steelhead rearing  to a point where mapping indicates gradients increase to 11%. Areas surveyed were rated as `r fpr::fpr_my_priority_info(sit = my_site, loc = 'us', col_pull = hab_value)`
-value for salmonid rearing and spawning. Although there was no dam structure located where indicated in provincial data layers, Dale Creek watershed does provide the community of Kispiox with domestic water so another dam may be present farther upstream.  The site was ranked as a
-`r fpr::fpr_my_priority_info(col_pull = priority)` priority for proceeding to design for replacement. Electrofishing upstream and downstream of the culvert could be conducted to to provide insight into fish community composition and density to help further inform prioritization.
+The BC freshwater atlas streamline indicates the stream gradient of the segment upstream of the FSR was 7.5% which coincides approximately with stream measurements.  This value is very near the 7.4% cutoff for modelling high value steelhead rearing habitat, so we estimated 0.8km of habitat suitable for steelhead rearing to a point where mapping indicates gradients increase to 11%. Areas surveyed were rated as `r fpr::fpr_my_priority_info(sit = my_site, loc = 'us', col_pull = hab_value)` value for salmonid rearing and spawning. Although there was no dam structure located where indicated in provincial data layers, Dale Creek watershed does provide the community of Kispiox with domestic water so another dam may be present farther upstream. 
+
+Following the 2022 survey, the site was ranked as `r fpr::fpr_my_priority_info(col_pull = priority)` priority for proceeding to design for replacement, with a recommendation to conduct electrofishing to help further inform prioritization. The site was revisited September 21, 2023 and electrofishing was conducted upstream and downstream of the crossing to provide insight into fish community composition and density. Cutthroat trout and rainbow trout fry and parr were captured upstream of the crossing, but there was no dolly varden presence upstream. Additionally, juvenile and adult cutthroat trout and rainbow trout were only captured downstream of the crossing. These results suggest that crossing `r as.character(my_site)` may be a barrier to certain life stages of cutthroat trout and rainbow trout, and is a barrier to dolly varden. Due to the crossing's proximity to the Kispiox River, migration is likely limited to times of high flows when the outlet drop would be reduced, which could explain why certain life stages were not captured upstream of the crossing.
```

Comment:
>Cutthroat trout and rainbow trout fry and parr were captured upstream of the crossing, but there was no dolly varden presence upstream. Additionally, juvenile and adult cutthroat trout and rainbow trout were only captured downstream of the crossing. These results suggest that crossing `r as.character(my_site)` may be a barrier to certain life stages of cutthroat trout and rainbow trout, and is a barrier to dolly varden.

well.... its pretty common for fish to be upstream of barriers even though they can't get through on their own. reason is that they persist there as "residents".  Interestingly there are even populations of fish that live upstream of huge waterfalls because the falls were created by the last ice age and they persist "resident" there as they have everything they need to live and reproduce. Some watersheds were joined at the top end by massive lakes during the end of the last ice age too so streams connected above barriers back in the day and they fish moved around.  Fun fact but makes interp tricky if our fish aren't anadromous.

Overall point is that I doubt any fish move through those pipes no matter the species or life stage.  The presence of dolly varden below and not above does deserve mention but it maaybe shouldn't imply the other species are migrating through - I'm going to think about it a bit more though.

The life stage interp is a bit funky b/c the bigger fish are downstream implying the smaller fish get through.... I would think the larger fish are likely d/s bc the habitat is better there for bigger fish (deep outlet pool)

Thoughtful shot at interpreting the results though.  Thank you and good work.

I will revise and finalize soon.

Reply:
sweet thanks for the feedback. I definitely need more experience interpreting results before I have this dialled so kinda sweet for me to give it a shot and then hear your side. I agree with the crossing being a barrier, and was a bit confused why there was small fish above and big fish below (would have assumed the opposite if anything) but thinking about resident fish above makes sense. Looking forward to getting more practice with interpreting results.